src/raft.h

```diff
@@ -765,6 +765,28 @@ struct raft_transfer; /* Forward declaration */
 
 struct raft_log;
 
+/**
+ * Strategy to compute the amount of trailing entries to keep.
+ * This can be tuned with raft_set_snapshot_trailing_strategy
+ */
+enum {
+	/**
+	 * The static strategy is to just keep a fixed amount of entries
+	 * in the log cache. The number of entries can be tuned with 
+	 * @raft_set_snapshot_trailing. This is the default.
+	 */
+	RAFT_TRAILING_STRATEGY_STATIC,
+	
+	/**
+	 * The dynamic strategy computes the number of entries to keep by 
+	 * comparing the size held by each entry to the size of the snapshot.
+	 * The idea behind this is that if the entries are too big, streaming
```

Comment:
> The idea behind this is that if the entries are too big, streaming the snapshot might be better.

Could you consider to rephrase it? This is a possible variant but it's not ideal:
If the amount of memory (on-disk or RAM) needed to store log entities exceeds the amount of memory for snapshot, streaming the snapshot is more efficient.